src/main/java/backend/resource/TurboIssue.java

```diff
@@ -338,6 +339,28 @@ public TurboIssue combineWithPullRequest(PullRequest pullRequest) {
         return Optional.empty();
     }
 
+    /**
+     * Matching is done by matching all words separated by space in query
```

Comment:
Is this an `AND` match or an `OR` match. Need to specify. Ideally, it should be the latter.